java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -165,6 +171,207 @@ public T set(String key, String value, SetOptions options) {
+     * only be cleared by commands that delete or overwrite the contents of <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/expire/">redis.io</a> for details.
+     * @param key - The key to set timeout on it.
+     * @param seconds - The timeout in seconds.
+     * @param expireOptions - The expire options.
+     * @return <code>true</code> if the timeout was set. <code>false</code> if the timeout was not
+     *     set. e.g. <code>key</code> doesn't exist, or operation skipped due to the provided
+     *     arguments.
+     */
+    public T expire(String key, long seconds, ExpireOptions expireOptions) {
+        ArgsArray commandArgs =
+                buildArgs(
+                        ArrayUtils.addAll(new String[] {key, Long.toString(seconds)}, expireOptions.toArgs()));
+
+        protobufTransaction.addCommands(buildCommand(Expire, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Sets a timeout on <code>key</code>. It takes an absolute Unix timestamp (seconds since January
+     * 1, 1970) instead of specifying the number of seconds. A timestamp in the past will delete the
+     * <code>key</code> immediately. After the timeout has expired, the <code>key</code> will
+     * automatically be deleted. If <code>key</code> already has an existing <code>expire</code> set,
+     * the time to live is updated to the new value. The timeout will only be cleared by commands that
+     * delete or overwrite the contents of <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/expireat/">redis.io</a> for details.
+     * @param key - The key to set timeout on it.
+     * @param unixSeconds - The timeout in an absolute Unix timestamp.
+     * @return <code>true</code> if the timeout was set. <code>false</code> if the timeout was not
+     *     set. e.g. <code>key</code> doesn't exist, or operation skipped due to the provided
+     *     arguments.
+     */
+    public T expireAt(String key, long unixSeconds) {
+        ArgsArray commandArgs = buildArgs(key, Long.toString(unixSeconds));
+
+        protobufTransaction.addCommands(buildCommand(ExpireAt, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Sets a timeout on <code>key</code>. It takes an absolute Unix timestamp (seconds since January
+     * 1, 1970) instead of specifying the number of seconds. A timestamp in the past will delete the
+     * <code>key</code> immediately. After the timeout has expired, the <code>key</code> will
+     * automatically be deleted. If <code>key</code> already has an existing <code>expire</code> set,
+     * the time to live is updated to the new value. The timeout will only be cleared by commands that
+     * delete or overwrite the contents of <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/expireat/">redis.io</a> for details.
+     * @param key - The key to set timeout on it.
```

Comment:
Documentation comments above if changed should also be applied for transactions.